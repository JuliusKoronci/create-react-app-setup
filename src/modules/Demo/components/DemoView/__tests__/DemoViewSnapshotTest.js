import React from 'react';
import DemoView from '../DemoView';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const wrapper = shallow(
	<DemoView
		list={[]}
		form={{
			title: 'test',
			desc: 'desc',
		}}
		actions={{
			'submit': () => {
			},
			'change': () => {
			},
			'deleteEntry': () => {
			},
		}}
	/>
);
describe('(Component) DemoView', () => {
	it('Should match snapshot', () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});

