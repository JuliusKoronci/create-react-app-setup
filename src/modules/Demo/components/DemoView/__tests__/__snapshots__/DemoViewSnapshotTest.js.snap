exports[`(Component) DemoView Should match snapshot 1`] = `
<div>
  <h2>
    List of entries
  </h2>
  <form
    onSubmit={[Function]}>
    <input
      onChange={[Function]}
      placeholder="title"
      type="text"
      value="test" />
    <input
      onChange={[Function]}
      placeholder="description"
      type="text"
      value="desc" />
    <button
      type="submit">
      create
    </button>
  </form>
</div>
`;
